pkg/sdk/symbols/info/info.go

```diff
@@ -105,7 +107,27 @@ func plugin_get_required_api_version() *C.char {
 	return (*C.char)(pRequiredAPIVersion.CharPtr())
 }
 
+func splitVersionString(version string) (string, string, string) {
+	nums := strings.Split(version, ".")
+	if len(nums) != 3 {
+		panic("Incorrect format. Expected: Semantic Versioning: X.Y.Z")
+	}
+	return nums[0], nums[1], nums[2]
+}
+
 func SetRequiredAPIVersion(apiVer string) {
+	requiredMajor, requiredMinor, requiredPatch := splitVersionString(apiVer)
```

Comment:
it seems that before doing this semver check we need to check for `apiVer != ""` because this will be the default value if the plugin creator doesn't specify anything, and it should be a legit value because in that the case the SDK will set the `C.get_default_required_api_version()`. Take as an example the `full` plugin in this repo if you check his [`Info()` method](https://github.com/falcosecurity/plugin-sdk-go/blob/c2c8a3e406da1828cd9a00c7dab39b2db0855286/examples/full/full.go#L93) you will see that it doesn't set any RequiredAPIVersion so the value will be ""

Reply:
Yes, This makes sense.